samples/app/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "gandi.translate-sample",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject",
+    "i18n": "react-translate-scripts"
+  },
+  "devDependencies": {
+    "babel-cli": "^6.18.0",
+    "babel-gettext-plugin": "^2.2.0",
```

Comment:
when a package delivers scripts, it should work out of box and so bring all the package it needs, am I right ?
(babel-cli is an exception since we let the user choose the version and prevent versions conflicts)

I think the "problem" is separation of concerns, actual react-translate isn't just a lib it's a toolbox with a react lib. May be it could be a meta-package witch provides some wrappers for all libs we support. 

(But that's obviously not pragmatic to start this enhancement)